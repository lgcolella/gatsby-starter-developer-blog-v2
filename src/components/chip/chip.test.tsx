/* Vendor imports */
import React from 'react'
import { mount } from 'enzyme'
/* App imports */
import Component from './index'

describe('Chip component', () => {
  test('matches snapshot', () => {
    let chip = mount(<Component>Text</Component>)
    expect(chip).toMatchSnapshot()
    let chipWithHoverEffect = mount(
      <Component hoverEffect={true}>Text</Component>
    )
    expect(chipWithHoverEffect).toMatchSnapshot()
    let chipWithClassName = mount(
      <Component className={'class'}>Text</Component>
    )
    expect(chipWithClassName).toMatchSnapshot()
    let chipWithHoverEffectAndClassName = mount(
      <Component hoverEffect={true} className={'class'}>
        Text
      </Component>
    )
    expect(chipWithHoverEffectAndClassName).toMatchSnapshot()
  })
})
