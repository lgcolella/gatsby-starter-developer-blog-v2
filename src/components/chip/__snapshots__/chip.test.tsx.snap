// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Chip component matches snapshot 1`] = `
<Chip>
  <label
    className="chip "
  >
    Text
  </label>
</Chip>
`;

exports[`Chip component matches snapshot 2`] = `
<Chip
  hoverEffect={true}
>
  <label
    className="chip_with_hover "
  >
    Text
  </label>
</Chip>
`;

exports[`Chip component matches snapshot 3`] = `
<Chip
  className="class"
>
  <label
    className="chip class"
  >
    Text
  </label>
</Chip>
`;

exports[`Chip component matches snapshot 4`] = `
<Chip
  className="class"
  hoverEffect={true}
>
  <label
    className="chip_with_hover class"
  >
    Text
  </label>
</Chip>
`;
